### PR TITLE
teraslice-cli await command for tjm and jobs

### PR DESCRIPTION
**Pull request by ciorg**
* await command blocks until status is reached for both tjm and jobs
* added stopped to terminal statuses in waitForStatus function in teraslice-client-js 
* jobs await parsing test
* added documentation
* other small fixes in docs

### CODE REVIEW

File: packages/teraslice-cli/src/cmds/jobs/await.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        try {
+            newStatus = await util.pRace(desiredStatus.map(
+                (status) => jobs.awaitStatus(
+                    status,
+                    jobs.config.args.id,
+                    jobs.config.args.timeout
+                )
+            ));
+        } catch (e) {
+            // @ts-ignore
```
Comment (peterdemartini) [inline packages/teraslice-cli/src/cmds/jobs/await.ts]: Why is there a `@ts-ignore` here?

File: packages/teraslice-cli/src/cmds/tjm/await.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        reply.green(`> job: ${jobFile.jobId} waiting for status ${desiredStatus.join(' or ')}`);
+
+        let newStatus;
+
+        try {
+            newStatus = await util.pRace(desiredStatus.map(
+                (status) => jobFuncs.waitForStatus(status, 5000, argv.timeout)
+            ));
+        } catch (e) {
+            // @ts-ignore
```
Comment (peterdemartini) [inline packages/teraslice-cli/src/cmds/tjm/await.ts]: Why is there a `@ts-ignore` here?

File: packages/teraslice-cli/src/cmds/jobs/await.ts (hunk 1 above)
Comment (ciorg) [reply packages/teraslice-cli/src/cmds/jobs/await.ts]: I get 
```
[ts] Property 'includes' does not exist on type 'ExecutionStatus[]
```

not sure why it doesn't think includes is an array property.

**Comment by ciorg** [comment]
addresses issue #954

**Pull request merged**